### PR TITLE
RC1 Releases

### PR DESCRIPTION
**Pull request by holgerd77**
See: #2561
Follow-up on #2832 

Release Candidate 1 versions preparing for the breaking release round for EthereumJS libraries planned in July 2023, part 2 of the release prepartion including library version updates, updated upstream dependency versions, README updates and docs updates.

Particularities:

- First `@ethereumjs/wallet` release within the new namespace (as RC1 as well)
- Client release included, no RC1 but normal release

### CODE REVIEW

**Review by acolytec3 — APPROVED** [review]
LGTM with these edits!

**Review by acolytec3 — DISMISSED** [review]
Reverting previous erroneous approval

**Comment by holgerd77** [comment]
Ok, PR is now up for a first round of reviews. 🙂 

So all README files are updated and all version numbers should be updated (please explicitly review this as well).

Not completely ready though, so I have not yet integrated the latest work, will do this on likely Monday and then open up for final approval.

Nevertheless if the current state of the PR would get a full review until the weekend so that we have this out of the way.

File: packages/util/src/index.ts [hunk 1]
```diff
@@ -43,16 +43,11 @@ export * from './bytes.js'
  */
 export * from './types.js'
 
-/**
- * Helper function for working with compact encoding
- */
-export * from './encoding.js'
```
Comment (jochem-brouwer) [inline packages/util/src/index.ts:49]: This is removed?

**Review by jochem-brouwer — COMMENTED** [review]
Comment: have reviewed so far the READMEs and the package.json's, (not the generated docs). Have to give this another round though.

**Comment by holgerd77** [comment]
Hi @scorbajio, sorry, I do not understand these two additional commits and I would therefore remove them.

**Comment by holgerd77** [comment]
@scorbajio if you do would want to do other changes around this please do not do on this PR but on a separate PR. If I am not totally overlooking something these changes should not be relevant for the release candidate releases though.

**Comment by holgerd77** [comment]
Ready for review. 🙂 

Some procedural notes: since I won't have time to work tomorrow afternoon and most of Wednesday a review in next 2 hours would help me in the form that I could then eventually already do the releases itself this afternoon and would have some more dedicated "marketing"/communication time tomorrow morning (I would do the "main" marketing along final releases though, but think here this also helps if we are doing some communication already to raise awareness and encourage people to try out).

Also: I am already doing some local udpates on th PR which I won't directly push (too tiny + incremental), since I am also cross-reading right now. Please therefore do not directly push updates on the branch if you review but only comment.

**Review by acolytec3 — APPROVED** [review]
LGTM

**Comment by holgerd77** [comment]
Just published the following releases on npm:
- `@ethereumjs/rlp` [v5.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Frlp%405.0.0-rc.1)
- `@ethereumjs/util` [v9.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Futil%409.0.0-rc.1)
- `@ethereumjs/common` [v4.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fcommon%404.0.0-rc.1)
- `@ethereumjs/tx` [v5.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%405.0.0-rc.1)
- `@ethereumjs/trie` [v6.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftrie%406.0.0-rc.1)
- `@ethereumjs/devp2p` [v6.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fdevp2p%406.0.0-rc.1)
- `@ethereumjs/wallet` [v2.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fwallet%402.0.0-rc.1)
- `@ethereumjs/statemanager` [v2.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fstatemanager%402.0.0-rc.1)
- `@ethereumjs/block` [v5.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblock%405.0.0-rc.1)
- `@ethereumjs/ethash` [v3.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fethash%403.0.0-rc.1)
- `@ethereumjs/blockchain` [v7.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblockchain%407.0.0-rc.1)
- `@ethereumjs/evm` [v2.0.0-rc.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fevm%402.0.0-rc.2)
- `@ethereumjs/vm` [v7.0.0-rc.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fvm%407.0.0-rc.1)
- `@ethereumjs/client` [v0.8.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fclient%400.8.0)

Additional note: second EVM release RC.2 was necessary due to missing StateManager dependency, have done locally, code will be submitted in a follow-up PR.

**Pull request merged**